examples/agent_rtm_example.py

```diff
@@ -8,6 +8,10 @@
 response = agent_rtm.start_chat(continuous=True)
 chat_id = response.payload.get('chat_id')
 thread_id = response.payload.get('thread_id')
+
+# Get all messages including the non-response messages (e.g. pushes). Returns a list
+incoming_chat_push = agent_rtm.ws.messages[0]
```

Comment:
Not sure if "all messages" and "Returns a list" is compatible with `[0]` used here. In that case I would consider rephrasing into sth like:
```suggestion
# Get `incoming_chat` push from all messages including the non-response messages (i.e. pushes)
incoming_chat_push = agent_rtm.ws.messages[0]
```